### PR TITLE
Change parameters defaults

### PR DESCRIPTION
**Pull request by alex-hse-repository**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

## Before submitting (checklist)

- [ ] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [ ] Did you read the [contribution guide](https://github.com/tinkoff-ai/etna-ts/blob/master/CONTRIBUTING.md)?
- [x] Did you check the code style? `make lint` (`poetry install -E style`).
- [x] Did you make sure to update the docs? We use Numpy format for all the methods and classes.
- [ ] Did you write any new necessary tests?
- [x] Did you check that your code passes the unit tests `pytest tests/` ? 
- [ ] Did you add your new functionality to the docs?
- [x] Did you update the [CHANGELOG](https://github.com/tinkoff-ai/etna-ts/blob/master/CHANGELOG.md)?

## Type of Change

- [ ] Examples / docs / tutorials / contributors update
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Improvement (non-breaking change which improves an existing feature)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Proposed Changes

  - Set the default values of hyperparameters as in original Catboost model

## PR review

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Closing issues

Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).

### CODE REVIEW

File: etna/models/catboost.py [hunk 1]
```diff
@@ -10,14 +10,17 @@
...
+        depth: int = 6,
+        learning_rate: float = None,
         logging_level: str = "Silent",
-        l2_leaf_reg: float = 6.735163225977638,
-        thread_count: int = 4,
+        l2_leaf_reg: float = 3,
+        thread_count: int = -1,
         **kwargs,
     ):
+        if l2_leaf_reg == 3:
```
Comment (julia-shenshina) [inline etna/models/catboost.py:21]: why?

File: etna/models/catboost.py [hunk 2]
```diff
@@ -161,22 +166,15 @@ def __init__(
             not affect the training.
             During the training one main thread and one thread for each GPU are used.
         """
-        self.iterations = iterations
```
Comment (Mr-Geekman) [inline etna/models/catboost.py:164]: Keep it, it is necessary for `__repr__` method.

**Comment by julia-shenshina** [comment]
update CHANGELOG please

**Comment by Mr-Geekman** [comment]
Update CHANGELOG.md

File: etna/models/catboost.py (hunk 1 above)
Comment (alex-hse-repository) [reply etna/models/catboost.py:21]: Setting this parameter turns off automatic learning_rate selection. See https://catboost.ai/docs/concepts/python-reference_parameters-list.html

File: etna/models/catboost.py [hunk 3]
```diff
@@ -10,14 +10,17 @@
 class _CatBoostModel:
     def __init__(
         self,
-        iterations: int = 100,
-        depth: int = 4,
-        learning_rate: float = 0.23,
+        iterations: int = 1000,
+        depth: int = 6,
+        learning_rate: float = None,
```
[the remark this answers is not in the document]
Comment (alex-hse-repository) [reply etna/models/catboost.py:15]: Done

File: etna/models/catboost.py (hunk 2 above)
Comment (alex-hse-repository) [reply etna/models/catboost.py:164]: Why do we have it only for CatBoostModelMultiSegment but not for CatBoostModelPerSegment?

Comment (Mr-Geekman) [reply etna/models/catboost.py]: We have it in all models except for CatBoost and SARIMAX. Then we should add it.

File: etna/models/catboost.py (hunk 1 above)
Comment (julia-shenshina) [reply etna/models/catboost.py:23]: but default CatboostRegressor's `l2_leaf_reg` value is 3 🤔 
and we try to provide default catboost behavior

File: etna/models/catboost.py (hunk 2 above)
Comment (julia-shenshina) [reply etna/models/catboost.py]: Add it to `CatBoostModelPerSegment` please 🙂

File: etna/models/catboost.py (hunk 1 above)
Comment (alex-hse-repository) [reply etna/models/catboost.py:23]: If you look at the __repr__ of the CatboostRegressor there are None values for all the parameters

File: etna/models/catboost.py (hunk 2 above)
Comment (alex-hse-repository) [reply etna/models/catboost.py]: Done

File: etna/models/catboost.py [hunk 4]
```diff
@@ -170,13 +184,13 @@ def __init__(
         self.kwargs = kwargs
         super(CatBoostModelMultiSegment, self).__init__()
         self._base_model = _CatBoostModel(
-            iterations=self.iterations,
```
[the remark this answers is not in the document]
Comment (julia-shenshina) [reply etna/models/catboost.py:173]: 🤔 
does it work?
I mean, `_CatboostModel` is a private field of `CatBoostModelMultiSegment`, not the base class; seems like `__repr__` couldn't find the args this way

File: etna/models/catboost.py (hunk 1 above)
Comment (julia-shenshina) [reply etna/models/catboost.py:23]: ...but docs say it is 3 🙂 
anyway, this `if` produces confusing behavior: you have no opportunity to set `l2_leaf_reg` with 3. 

if default catboost `l2_leaf_reg` should be None, add None as a default value in our `_CatBoostModel.__init__` method please

**Review by julia-shenshina — APPROVED** [review]

**Review by iKintosh — APPROVED** [review]

**Pull request merged**